test/conformance/README.md

```diff
@@ -8,4 +8,11 @@ In the future, when all bugs are fixed, and the tests pass,
 this solution will no longer be necessary.
 When you fix any test, the match file must be updated
 Empty match files indicate that there are no failing tests
-in a particular group for the corresponding adapter.
\ No newline at end of file
+in a particular group for the corresponding adapter.
+
+## How to limit the test devices count
+
+To limit how many devices you want to run the CTS on,
+use CMake option UR_TEST_DEVICES_COUNT. If you want to run
```

Comment:
could we make it a runtime option instead of a CMAKE one? imagine the case where users have the binaries for the CTS, and have different types of systems (single CPU, 4xGPU, CPU + 2xGPU, etc). They would have to build the binaries for each configuration.

Reply:
thanks @szadam . could you share why it couldn't be done as part of this PR? Main reason I'm asking is because I have already run into this problem, where I was running the tests in a multi-device system, and it took me a while to figure out the issue was that the tests were built only for 1 device, so I would expect more people to run into the same problem, so addressing sooner would be great.

Reply:
if you run the tests directly through their binary, you can use the `--test_devices_count 1`  cli option that @szadam is adding. If you are using ctest, this patch will automatically reduce the number of devices it will run with to 1. You don't need to set the cmake option.

There's another patch that @szadam is working on, #992, that will solve the broader problem of devices/platforms having the same names.

Reply:
I think what @jandres742 is proposing is implemented for OpenCL patch (here: https://github.com/oneapi-src/unified-runtime/pull/993/files#diff-d1f50b1ed696f085259b2a9c7d274ae3414700a1cea60f9945677cdae39e25cfR186) and it looks neat as well. The CMake option, though, has, I think, the added value (as Piotr mentioned) that it sets the default value of 1 and still can be overridden with the CLI param.

BTW, Adam has a new version (still on a private branch; to be polished): https://github.com/szadam/unified-runtime/pull/15/commits/ef742824118957136ecfc434f84feb446a366292#diff-d1f50b1ed696f085259b2a9c7d274ae3414700a1cea60f9945677cdae39e25cf

Reply:
thanks @pbalcer and @lukaszstolarczuk for the details.

Reply:
Sorry for the late response. I am preparing a new solution/version as Lukas said. I will push it today.